src/tcgwars/logic/impl/gen7/ShinyVault.groovy

```diff
@@ -0,0 +1,398 @@
+	LYCANROC_GX_SV66 ("Lycanroc-GX", SV66, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _FIGHTING_]),
+	LYCANROC_GX_SV67 ("Lycanroc-GX", SV67, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _FIGHTING_]),
+	BUZZWOLE_GX_SV68 ("Buzzwole-GX", SV68, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _FIGHTING_]),
+	UMBREON_GX_SV69 ("Umbreon-GX", SV69, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _DARKNESS_]),
+	CHARMELEON_SV7 ("Charmeleon", SV7, Rarity.HOLORARE, [POKEMON, EVOLUTION, STAGE1, _FIRE_]),
+	DARKRAI_GX_SV70 ("Darkrai-GX", SV70, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _DARKNESS_]),
+	GUZZLORD_GX_SV71 ("Guzzlord-GX", SV71, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _DARKNESS_]),
+	SCIZOR_GX_SV72 ("Scizor-GX", SV72, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _METAL_]),
+	KARTANA_GX_SV73 ("Kartana-GX", SV73, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _METAL_]),
+	STAKATAKA_GX_SV74 ("Stakataka-GX", SV74, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _METAL_]),
+	GARDEVOIR_GX_SV75 ("Gardevoir-GX", SV75, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _FAIRY_]),
+	SYLVEON_GX_SV76 ("Sylveon-GX", SV76, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _FAIRY_]),
+	ALTARIA_GX_SV77 ("Altaria-GX", SV77, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _DRAGON_]),
+	NOIVERN_GX_SV78 ("Noivern-GX", SV78, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _DRAGON_]),
+	SILVALLY_GX_SV79 ("Silvally-GX", SV79, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _COLORLESS_]),
+	ALOLAN_VULPIX_SV8 ("Alolan Vulpix", SV8, Rarity.HOLORARE, [POKEMON, BASIC, _WATER_]),
+	DRAMPA_GX_SV80 ("Drampa-GX", SV80, Rarity.ULTRARARE, [POKEMON, BASIC, POKEMON_GX, _COLORLESS_]),
+	AETHER_FOUNDATION_EMPLOYEE_SV81 ("Aether Foundation Employee", SV81, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	CYNTHIA_SV82 ("Cynthia", SV82, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	FISHERMAN_SV83 ("Fisherman", SV83, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	GUZMA_SV84 ("Guzma", SV84, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	HIKER_SV85 ("Hiker", SV85, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	LADY_SV86 ("Lady", SV86, Rarity.ULTRARARE, [TRAINER, SUPPORTER]),
+	AETHER_PARADISE_CONSERVATION_AREA_SV87 ("Aether Paradise Conservation Area", SV87, Rarity.SECRET, [TRAINER, STADIUM]),
+	BROOKLET_HILL_SV88 ("Brooklet Hill", SV88, Rarity.SECRET, [TRAINER, STADIUM]),
+	MT_CORONET_SV89 ("Mt. Coronet", SV89, Rarity.SECRET, [TRAINER, STADIUM]),
+	WOOPER_SV9 ("Wooper", SV9, Rarity.HOLORARE, [POKEMON, BASIC, _WATER_]),
+	SHRINE_OF_PUNISHMENT_SV90 ("Shrine of Punishment", SV90, Rarity.SECRET, [TRAINER, STADIUM]),
+	TAPU_BULU_GX_SV91 ("Tapu Bulu-GX", SV91, Rarity.SECRET, [POKEMON, BASIC, POKEMON_GX, _GRASS_]),
+	TAPU_FINI_GX_SV92 ("Tapu Fini-GX", SV92, Rarity.SECRET, [POKEMON, BASIC, POKEMON_GX, _WATER_]),
+	TAPU_KOKO_GX_SV93 ("Tapu Koko-GX", SV93, Rarity.SECRET, [POKEMON, BASIC, POKEMON_GX, _LIGHTNING_]),
+	TAPU_LELE_GX_SV94 ("Tapu Lele-GX", SV94, Rarity.SECRET, [POKEMON, BASIC, POKEMON_GX, _PSYCHIC_]),
```

Comment:
Remove the trailing comma here, otherwise CI will complain

also bring the semi colon from line 148 up so that it replaces the comma here